language-support/java/bindings-rxjava/src/test/scala/com/daml/ledger/rxjava/grpc/CommandClientImplTest.scala

```diff
@@ -50,22 +49,21 @@ class CommandClientImplTest
 
   behavior of "[2.1] CommandClientImpl.submitAndWait"
 
-  it should "send the given command to the Ledger" in {
+  it should "send the given command to the Ledger with new params" in {
```

Comment:
```suggestion
  it should "send the given command to the Ledger" in {
```

It doesn't look like the tests are in another class and this not going to be "new" very soon, so I'd keep it simple. 🙂